examples/bicop/src/CMakeLists.txt

```diff
@@ -1,9 +1,5 @@
-# Include header files
-include_directories(${external_includes})
```

Comment:
Sorry, asking for my personal understanding: why don't we need the external includes anymore? Are they automatically included by the "dark magic" above?

Reply:
yes, they are automatically added when you link to vinecopulib